src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -50,6 +50,9 @@ function LtiConfigForm({
       <Form ref={formRef} onSubmit={handleSubmit}>
         <h3 className="mb-3">{title}</h3>
         <p>{intl.formatMessage(messages.formInstructions)}</p>
+        {app.additionalMessages && app.additionalMessages.map(msg => (
```

Comment:
I think we should just have a single message for this, and leave scope open for future messages in the API.

Reply:
I've made a [comment on the backend PR](https://github.com/edx/edx-platform/pull/28092#discussion_r667500599) describing my rationale but let me know if you still want it changed.